packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -216,13 +242,15 @@ export class RouterliciousOrdererRestWrapper extends RouterliciousRestWrapper {
                         documentId,
                         refreshToken,
                     );
-                    return `Basic ${ordererToken.jwt}`;
+
+                    return ordererToken;
                 }
             );
         };
 
         const restWrapper = new RouterliciousOrdererRestWrapper(
-            logger, rateLimiter, getAuthorizationHeader, useRestLess, baseurl);
+            logger, rateLimiter, fetchOrdererToken, getAuthorizationHeader, useRestLess, baseurl);
+
         try {
```

Comment:
note if following comment about cleaning up undefined cases, this `try catch` could just wrap a `return RouterliciousRestWrapper.load(...)` call